{"name": "登录失败场景", "status": "passed", "description": "登录失败场景", "attachments": [{"name": "stdout", "source": "f302afae-2e65-41de-923f-0d3620341af8-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "browser_name", "value": "'chromium'"}, {"name": "username", "value": "'py11'"}, {"name": "password", "value": "'12345678'"}, {"name": "title", "value": "'输入错误的账号'"}], "start": 1756956337480, "stop": 1756956338105, "uuid": "f62c5abd-6435-42db-84c3-a3d2cf213065", "historyId": "1093d470bbc4a1db0cdc77f682d441c0", "testCaseId": "0171dfb4726b6d084b4a08882eabf7ca", "fullName": "cases.test_login.TestLogin#test_login_error", "labels": [{"name": "feature", "value": "登录功能"}, {"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Albert"}, {"name": "thread", "value": "20752-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.test_login"}], "titlePath": ["cases", "test_login.py", "TestLogin"]}